src/app/pages/BridgeWithdrawPage/components/ReviewStep/index.tsx

```diff
@@ -0,0 +1,186 @@
+    chain as any,
+    targetChain as any,
+    currentAsset,
+  );
+
+  const valid = useMemo(() => {
+    const bnAmount = bignumber(amount || '0');
+    const bnBalance = bignumber(balance.value || '0');
+    return (
+      !limitsLoading &&
+      !balance.loading &&
+      bnBalance.greaterThanOrEqualTo(bnAmount) &&
+      bnAmount.greaterThan(0) &&
+      bnAmount.greaterThanOrEqualTo(limits.returnData.getMinPerToken) &&
+      bnAmount.lessThanOrEqualTo(limits.returnData.getMaxTokensAllowed) &&
+      bignumber(limits.returnData.dailyLimit).greaterThanOrEqualTo(
+        bnAmount.add(limits.returnData.spentToday),
+      )
+    );
+  }, [
+    amount,
+    balance.loading,
+    balance.value,
+    limits.returnData.dailyLimit,
+    limits.returnData.getMaxTokensAllowed,
+    limits.returnData.getMinPerToken,
+    limits.returnData.spentToday,
+    limitsLoading,
+  ]);
+
+  return (
+    <div className="tw-flex tw-flex-col tw-items-center tw-mw-320">
+      <div className="tw-mb-20 tw-text-2xl tw-text-center tw-font-semibold">
+        Review {currentAsset.symbol} Withdraw
+      </div>
+      <div className="tw-mw-320">
+        <Table className="tw-mx-auto">
+          <tbody>
+            <tr>
+              <td>Date/Time:</td>
+              <td>{new Date().toLocaleDateString()}</td>
+            </tr>
+            <tr>
+              <td>From:</td>
+              <td>{currentNetwork?.name}</td>
+            </tr>
+            <tr>
+              <td>To:</td>
+              <td>{network?.name}</td>
+            </tr>
+            <tr>
+              <td>Token:</td>
+              <td>
+                {currentAsset?.symbol} -&gt; {asset?.symbol}
+              </td>
+            </tr>
+            <tr>
+              <td>Amount:</td>
+              <td>
+                {toNumberFormat(
+                  currentAsset.fromWei(amount),
+                  currentAsset.minDecimals,
+                )}
+              </td>
+            </tr>
+            <tr>
+              <td>Receiver:</td>
+              <td>
+                <a
+                  href={network.explorer + '/address/' + receiver}
+                  rel="noreferrer noopener"
+                  target="_blank"
+                >
+                  {prettyTx(receiver)}
+                </a>
+              </td>
+            </tr>
+            <tr>
+              <td>Bridge Fee:</td>
+              <td>
+                {toNumberFormat(
+                  currentAsset.fromWei(limits.returnData.getFeePerToken),
+                  currentAsset.minDecimals,
+                )}{' '}
+                {currentAsset.symbol}
+              </td>
+            </tr>
+          </tbody>
+        </Table>
+
+        <Button
+          className="tw-mt-20 tw-w-full"
+          text={'Confirm Withdraw'}
+          disabled={!valid || tx.loading}
+          loading={tx.loading}
+          onClick={handleSubmit}
+        />
+      </div>
+    </div>
+  );
+}
+
+const Table = styled.table`
```

Comment:
I have seen this in multiple files, probably extract it to some `styled.ts` file and use it on every occurrence.